content/welcoming-the-rust-innovation-lab.md

```diff
@@ -0,0 +1,21 @@
++++
+path = "2025/09/03/welcoming-the-rust-innovation-lab"
+title = "Welcoming the Rust Innovation Lab"
+authors = ["Carol Nichols"]
+
+[extra]
+team = "the Rust Foundation Project Directors"
+team_url = "https://www.rust-lang.org/governance/teams/launching-pad#team-foundation-board-project-directors"
++++
+
+TL;DR: [Rustls](https://github.com/rustls/rustls) is the inaugural project of the [Rust Innovation Lab](https://rustfoundation.org/rust-innovation-lab), which is a new home for Rust projects under the Rust Foundation.
+
+At the Rust Foundation's August meeting, the Project Directors and the rest of the Rust Foundation board voted to approve [Rustls](https://github.com/rustls/rustls) as the first project housed under the newly formed Rust Innovation Lab. Prior to the vote, the Project Directors consulted with the Leadership Council who confirmed the Project's support for this initiative.
+
+The Rust Innovation Lab (RIL) is designed to provide support for funded Rust-based open source projects from the Rust Foundation in the form of governance, legal, networking, marketing, and administration, while keeping the technical direction solely in the hands of the current maintainers. As with the other work of the Rust Foundation (e.g. its many existing [initiatives](https://rustfoundation.org/initiatives/)), the purpose of the RIL is to strengthen the Rust ecosystem generally.
+
+The Foundation has been working behind the scenes to establish the Rust Innovation Lab, which includes setting up infrastructure under the Foundation to ensure smooth transition for Rustls into RIL. More details are available in [the Foundation's announcement](https://rustfoundation.org/media/rust-foundation-launches-rust-innovation-lab-with-rustls-as-inaugural-project/) and on [the Rust Innovation Lab's page](https://rustfoundation.org/rust-innovation-lab).
```

Comment:
https://rustfoundation.org/rust-innovation-lab/ is password protected currently.

Reply:
Yep, I just reloaded and it's live now.